godot-kotlin/godot-library/build.gradle.kts

```diff
@@ -6,13 +6,28 @@ plugins {
 
 //TODO: this needs to be properly configured! This is just a basic setup to be able to implement the annotations
 kotlin {
-    linuxX64()
-    mingwX64()
-    macosX64()
+    macosX64("macos")
+    linuxX64("linux")
+    mingwX64("windows")
+
+    val internal = sourceSets.create("nativeInternal")
```

Comment:
please don't just use the word `internal`. That's a keyword ;-)
Make it internalSourceSet or nativeInternal or internalNative :-)